test/e2e/testing-manifests/storage-csi/mock/csi-mock-driver.yaml

```diff
@@ -29,7 +29,7 @@ spec:
             - mountPath: /csi
               name: socket-dir
         - name: driver-registrar
-          image: k8s.gcr.io/sig-storage/csi-node-driver-registrar:v1.3.0
+          image: k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.0.1
```

Comment:
https://github.com/kubernetes/k8s.io/blob/798f7d6ac63d50dc79e96f5915644fad1f2e851b/k8s.gcr.io/images/k8s-staging-sig-storage/images.yaml#L15
I checked the image list here and find that v2.1.0 is the latest one.

Reply:
Darn, a typo. I was looking at the list of images at https://console.cloud.google.com/gcr/images/k8s-artifacts-prod/US/sig-storage/csi-node-driver-registrar?gcrImageListsize=30 and then swapped digits. :cry:

Reply:
Thanks for checking. Fixed now.